Example/Podfile

```diff
@@ -0,0 +1,58 @@
+source 'https://github.com/CocoaPods/Specs.git'
+project 'Example.xcodeproj'
+platform :ios, '10.0'
+
+# ignore all warnings from all pods
+inhibit_all_warnings!
+
+use_frameworks!
+
+def sdk_pods
+    pod 'Locksmith', '4.0.0'
+    pod 'Moya', '11.0.2'
+    pod 'ReactiveSwift', '3.1.0'
+    pod 'WebRTC', '63.11.20455'
+    pod 'Socket.IO-Client-Swift', '9.0.1'
+    pod 'ObjectMapper', '3.3.0'
+    pod 'SwiftPhoenixClient', '0.9.1'
+    pod 'Macaw', '0.9.3'
```

Comment:
Why all of these are needed? I though they are part of the SDK?

Reply:
SDK is only linking this libraries dynamically in runtime
https://theswiftdev.com/2018/01/25/deep-dive-into-swift-frameworks/

Reply:
Ok, will developers know which libraries to install from the documentation?

Reply:
These will automatically be installed with the SDK, removed them from the list.